camelot/view/controls/editors/floateditor.py

```diff
@@ -181,7 +180,7 @@ def __init__(self, parent,
         layout.setSpacing(0)
         layout.addWidget(spinBox)
         layout.addWidget(self.calculatorButton)
-        self.add_actions(actions, layout)
+        self.add_actions(kwargs['action_routes'], layout)
```

Comment:
Waarom deze niet gewoon als argument meegeven in de constructor als deze altijd gedefinieerd moet zijn? Want op deze manier gaat het toch falen als het niet wordt meegegeven?